### PR TITLE
Issue #SB-24310 feat: Tagging org and target framework values for assets created under a target collection

### PR DESCRIPTION
**Pull request by rajnishdargan**
<table>
<td>
  <a href="https://project-sunbird.atlassian.net/browse/SB-24310" title="SB-24310" target="_blank">StorySB-24310</a>  Tagging org and target framework values for assets created under a target collection
  </td></table>
  <br />
 

Please include a summary of the change and which issue is fixed. Please also include relevant motivation and context. List any dependencies that are required for this change.

### Type of change

Please choose appropriate options.

- [ ] Bug fix (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)
- [ ] This change requires a documentation update

### How Has This Been Tested?

Please describe the tests that you ran to verify your changes in the below checkboxes. Provide instructions so we can reproduce. Please also list any relevant details for your test configuration

- [ ] Ran Test A
- [ ] Ran Test B

**Test Configuration**:
* Software versions: Angular 8, Nodejs 12.16.1
* Hardware versions:

### Checklist:

- [ ] My code follows the style guidelines of this project
- [ ] I have performed a self-review of my own code
- [ ] I have commented my code, particularly in hard-to-understand areas
- [ ] I have made corresponding changes to the documentation
- [ ] My changes generate no new warnings
- [ ] I have added tests that prove my fix is effective or that my feature works
- [ ] New and existing unit tests pass locally with my changes
- [ ] Any dependent changes have been merged and published in downstream modules

### CODE REVIEW

**Review by pallakartheekreddy — APPROVED** [review]

File: src/app/client/src/app/modules/sourcing/components/content-uploader/content-uploader.component.ts [hunk 1]
```diff
@@ -155,6 +156,12 @@ export class ContentUploaderComponent implements OnInit, AfterViewInit, OnDestro
     this.pageStartTime = Date.now();
     this.setTelemetryStartData();
     this.helperService.initialize(this.programContext);
+    const targetFWIds = _.get(this.sessionContext.targetCollectionFrameworksData, 'targetFWIds');
+    if (targetFWIds) {
+      if (!_.isUndefined(targetFWIds)) {
```
Comment (venkateshwarans) [inline src/app/client/src/app/modules/sourcing/components/content-uploader/content-uploader.component.ts:161]: can we remove one `if`

File: src/app/client/src/app/modules/core/services/framework/framework.service.ts [hunk 2]
```diff
@@ -124,4 +131,89 @@ export class FrameworkService {
...
+            'objectType': 'Category',
+            'status': ['Live']
+        },
+        'exists': ['orgIdFieldName', 'targetIdFieldName'],
+        'fields': ['code', 'identifier', 'orgIdFieldName', 'targetIdFieldName'],
+        'limits': 300
+      }
+    }
+    };
+
```
Comment (venkateshwarans) [inline src/app/client/src/app/modules/core/services/framework/framework.service.ts:189]: @pallakartheekreddy can you just verify this search request for framework categories.

**Review by venkateshwarans — COMMENTED** [review]
check comments. rest looks good to me.

File: src/app/client/src/app/modules/sourcing/components/content-uploader/content-uploader.component.ts (hunk 1 above)
Comment (rajnishdargan) [reply src/app/client/src/app/modules/sourcing/components/content-uploader/content-uploader.component.ts]: removed two if checks, using one if only now

**Pull request merged**